debian/update-motd.d/10-help-text

```diff
@@ -20,16 +19,6 @@
 #    with this program; if not, write to the Free Software Foundation, Inc.,
 #    51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
 
-[ -r /etc/lsb-release ] && . /etc/lsb-release
+echo "\nNote: This login is supported for developers and field service only."
+echo "Documentation: https://github.com/endlessm/eos-documentation/wiki\n"
```

Comment:
I would remove this banner completely. There's currently no way for an user to install the openssh server without running `eos-convert-system`; but if we ever add a way for the user to install it (conceivable since we ship a terminal already) the banner would be incorrect.